rustdoc.el

```diff
@@ -76,103 +96,68 @@ This is useful if you want to search for the name of a struct, enum or trait."
                       nil
                       nil
                       (thing-at-point 'symbol))))
-  (let ((helm-ag-base-command "rg  --smart-case --no-heading --color=never --line-number")
+  (let ((helm-ag-base-command "rg -L --smart-case --no-heading --color=never --line-number")
+        (search-directory (concat (file-name-as-directory (lsp-workspace-root))
+                                  rustdoc-local-directory))
         (regex (if current-prefix-arg
                    (progn
                      (setq current-prefix-arg nil)
                      "^\\* [^-]\*")
                  "\\* [^-]\*")))
-    (helm-ag rustdoc-search-directory (concat regex search-term))))
-
-
-;;;###autoload
-(defun rustdoc-convert-directory (&optional directory)
-  "Convert all .html files in DIRECTORY and its subdirectories to .org.
-Place the files in `rustdoc-search-directory`
-If DIRECTORY is not given, prompts user to select directory."
-  (interactive)
-  (make-directory rustdoc-search-directory t)
-  (let ((dir (if directory
-                 directory
-               (read-directory-name "Directory with rust html docs (for std: ~/.rustup/toolchains/<arch>/share/doc/rust/html/std): "))))
-    (rustdoc--get-filter)
-    (dolist (file (directory-files-recursively dir ".html"))
-      (if (with-temp-buffer
-            (insert-file-contents file)
-            (< 10 (count-lines (point-min)
-                               (point-max))))
-          (progn
-            (rustdoc-convert-file dir file))
-        (message "Ignoring conversion of %s as it is probably a redirect"
-                 file)))
-    (message "Batch conversion done!")))
+    (helm-ag search-directory (concat regex search-term))))
```

Comment:
Will this search both std and package-specific documentation?

Reply:
Oh I see now. It's creating a symlink to the std directory if you have package specific documentation?
Smart. What happens if you do not have any package specific documentation?
(Asking since I can't run it at the moment, see below)

Reply:
Fixed a bug with the symlink as of 8197f43, if there is no package specific documentation, it will just search `std` via that symlink.